libcloudforensics/providers/gcp/internal/project.py

```diff
@@ -35,19 +35,19 @@ class GoogleCloudProject:
     default_zone: Default zone to create new resources in.
 
   Example use:
-    gcp = GoogleCloudProject("your_project_name", "us-east")
+    gcp = GoogleCloudProject("your_project_name", "us-east1-b")
     gcp.compute.ListInstances()
   """
 
   def __init__(self,
                project_id: str,
-               default_zone: Optional[str] = None) -> None:
+               default_zone: Optional[str] = 'us-central1-f') -> None:
```

Comment:
If there's a default value for a parameter then the `Optional` type hint keyword is to be dropped: `str: 'us-central1-f`